### ISSUE TITLE
Zurb Foundation 5 framework for Magento ?

### ISSUE THREAD

**Issue by topstarnetwork** [nandroid/MagentoFoundation#16]
Hello, is there any Zurb Foundation 5 framework for Magento ?

**Comment by versedi** [comment]
http://foundation.zurb.com/develop/resources.html

# Magento
- Waterlee Boilerplat by Jake Sharp  F5 https://github.com/zeljkoprsa/waterlee-boilerplate
- Magento & Foundation by Nandroid  F3 https://github.com/nandroid/MagentoFoundation

I'm working with nandroid's version but updated to F5 manually.

**Comment by topstarnetwork** [comment]
Let me answer the 2 url links you gave me:
as far as waterlee is concerned it is said it is not the way to go here http://awesomescreenshot.com/032403wrfe  original link here http://www.demacmedia.com/design-user-experience/making-magento-responsive-using-foundation-part-2/

and as far as second link you gave me this is for Magento & Foundation 3 and not Foundation 5

**Comment by topstarnetwork** [comment]
Oh, sorry I did not see you worte 'I updated it manually,
would you have 1 zip file to send me please?
Regards, Heather

**Comment by versedi** [comment]
Sorry I cannot as I don't have original files and it's very late project that I'm doing in my full part job. 

But it's not so hard as it seems to be – just replace Foundation's files, replace F3 codes into F5 codes with text editor, check layout/*.xml and template/page/html/head.php for name changes (there are some in js files if I remember correctly). Everything should run smoothly if there will not be any problems with jQuery conflict – be aware of that and check your network tab in development tool for list of downloaded js files. 

Maybe during christmas I'll have some time to play with it and if lucky pull some request into nandroid's branch with F5 or do my own based on theirs solutions and mine. Besides I'd really like to see preprocessor implemented into it. Question is – is it worth it as Magento 2 is coming?

**Comment by topstarnetwork** [comment]
Thanks for your help!

**Comment by nandroid** [comment]
Hi, I'm nandroid!!
I didn't update the Foundation framework to 5 because I did't work with Magento lately.
But if somebody helps me we can do it.
Now I'm with Foundation for Apps.

**Comment by topstarnetwork** [comment]
Well I am not a it literate, no coder at all.
In this case how can we do?
Is the upgrade something that takes time?

**Comment by versedi** [comment]
For simple CSS version we should just follow steps something like I've mentioned above. In my humble opinion implementing Sass would be much more helpful as Magento theme making in pure CSS is really time consuming.

**Comment by topstarnetwork** [comment]
Right, we need definatly Sass so how are we going to do as I am not a software developer?????
Magento 2 seems emplementing saas if I am not mistaking new release  https://wiki.magento.com/display/MAGE2DOC/Magento+2+Development+Updates 

In any case what do we need to get it started aspecially as you ar ethe only developer????
Regards, Heather

**Comment by versedi** [comment]
Okay guys, I've started something but for now it's a lot of mess and I'd like some help with things like setting submodule in git for foundation auto update. 

https://github.com/versedi/Found5Mage

For now it's just connected with app.css, I don't know if you'll be able to run _grunt_ to process scss files. that's a thing someone could check as I am not really familiar with grunt commands on that level.

**Comment by topstarnetwork** [comment]
I wich I could help you but I am no IT literate. I hope someone will assist you.
